lib/build/tasks.js

```diff
@@ -45,11 +45,6 @@ function generateAssetsSync ({ verbose } = {}) {
   generateCss(appSassPath, appCssPath, appSassOptions)
   generateCss(libSassPath, appCssPath, libSassOptions)
   if (verbose) process.stdout.write('done\n')
-
-  if (verbose) process.stdout.write('copying assets...')
-  copyAssets(libAssetsPath, path.join(paths.public, '_kit'))
-  copyAssets(paths.assets, paths.public)
-  if (verbose) process.stdout.write('done\n')
```

Comment:
If we're no longer copying assets we can get rid of the `watchAssets` and `deleteAssets` functions as well.

Reply:
We should also make sure the related changes in `server.js` are in the same commit as these changes, currently they are spread across two commits, [731bd0c (Made progress with the tests (and changes to production highlighted by the tests).)](https://github.com/alphagov/govuk-prototype-kit/commit/731bd0c35d13dfe553d7288c99ec42902c7badea) and [02973418 (Cypress tests)](https://github.com/alphagov/govuk-prototype-kit/pull/1533/commits/02973418d0dbc40ca3d90f28ff7953d0d0f9372e).

Reply:
I'm nervous about getting that deep into the history here alone, I'm happy to pair on it.

Reply:
We squashed everything.